src/pineko/evolve.py

```diff
@@ -148,28 +151,60 @@ def evolve_grid(
         renormalization scale variation
     xif : float
         factorization scale variation
-    alphas_values : None or list
-        values of strong coupling used to collapse grids
     assumptions : str
         assumptions on the flavor dimension
     comparison_pdf : None or str
         if given, a comparison table (with / without evolution) will be printed
     """
     _x_grid, _pids, mur2_grid, _muf2_grid = grid.axes()
```

Comment:
```suggestion
    x_grid, _pids, mur2_grid, _muf2_grid = grid.axes()
```
and accordingly below (now it is used)